logos/brand-guidelines.md

```diff
@@ -0,0 +1,80 @@
+# Resources and Guidelines for TiDB Community
+
+These resources and guidelines were designed to assist the TiDB Community to effectively and consistently build and protect the community’s brand 
+through visual communication.
+
+# TiDB Logo
+
+## Seal+Signature(formal)
+
+![seal signature-normal](https://user-images.githubusercontent.com/6755791/33934949-4223d32c-e035-11e7-9884-2dd67a024319.png)
+
+DOWNLOAD LOGO (EPS)⬇️
+
+DOWNLOAD LOGO (PNG)⬇️
+
+## Seal+Signature(black)
+
+![seal signature-black](https://user-images.githubusercontent.com/6755791/33934948-41e3c8b8-e035-11e7-9fa8-38ae140fb1c1.png)
+
+DOWNLOAD LOGO (EPS)⬇️
+
+DOWNLOAD LOGO (PNG)⬇️
+
+## Colors for Seal+Signature
+
+![color-guide-blue](https://user-images.githubusercontent.com/6755791/33934941-3eaa73b8-e035-11e7-9844-df51234f9eca.png)
+
+**Red**
+CMYK: 0-100-80-0 ; RGB: 228-2-45 ; Hex: E4022D
+
+**Blue**
+CMYK: 95-83-42-38 ; RGB: 30-47-79 ; Hex: 1E2F4F
+
+**Black**
+CMYK: 0-0-0-100 ; RGB: 0-0-0 ; Hex: 000000
+
+## Seal(formal)
+
+![seal-guide](https://user-images.githubusercontent.com/6755791/33933049-8a6827ec-e02f-11e7-9dde-ac1e561a6481.png)
+
+DOWNLOAD LOGO (EPS)⬇️
+
+DOWNLOAD LOGO (PNG)⬇️
+
+## Seal(1 Color with Background)
+
+![seal-guide-bg](https://user-images.githubusercontent.com/6755791/33933436-ac2dc8f4-e030-11e7-8df5-b442e82492b1.png)
+
+DOWNLOAD LOGO (EPS)⬇️
+
+DOWNLOAD LOGO (PNG)⬇️
+
+## Colors for Signature
+
+![color-guide](https://user-images.githubusercontent.com/6755791/33932236-54364034-e02d-11e7-9e28-134bad04227f.png)
+
+**Red**
+CMYK: 0-100-80-0 ; RGB: 228-2-45 ; Hex: E4022D
+
+**Cool Grey**
+CMYK: 48-36-24-66 ; RGB: 77-79-83 ; Hex: 4D4F53
+
+**Black**
+CMYK: 0-0-0-100 ; RGB: 0-0-0 ; Hex: 000000
+
+# Do these awesome things
+
+- √ Use the TiDB logo to link to TiDB and PingCAP
+- √ Use the Mark in social buttons to link to your TiDB project
+- √ Use the TiDB logo to advertise that your product use TiDB as servise
+- √ Use the TiDB logo in a blog post or news article about TiDB
+
+# Please don’t do these things
+
+- X Use the TiDB logo for your application’s icon
```

Comment:
-> Use the TiDB logo as your application's icon

Reply:
thanks a lot